docs/sources/mimir/operators-guide/mimir-runbooks/_index.md

```diff
@@ -476,6 +476,25 @@ How to **investigate**:
 
 - Look for details in the ingester logs
 
+### MimirIngesterInstanceHasNoTenants
+
+This alert fires when an ingester instance doesn't own any tenants and is therefore idling.
+
+How it **works**:
+
+- Ingesters join a hash ring that facilitates per-tenant request sharding across ingester replicas.
+- Distributors shard requests that belong to an individual tenant across a subset of ingester replicas. The number of replicas used per tenant is determined by the `-distributor.ingestion-tenant-shard-size` or the `ingestion_tenant_shard_size` limit.
+- When the tenant shard size is lower than the number of ingester replicas, some ingesters might not receive requests for any tenants.
+- This is more likely to happen in Mimir clusters with a lower number of tenants.
+
+How to **fix** it:
+
+Choose one of three options:
+
+- Decrease the number of ingester replicas to match the highest number of shards per tenant.
```

Comment:
Scaling down ingesters is a tricky procedure. It's not a "quick fix". https://grafana.com/docs/mimir/latest/operators-guide/run-production-environment/scaling-out/#scaling-down-ingesters

Perhaps we should suggest next two items first.